.github/workflows/test-installers.yml

```diff
@@ -181,6 +181,15 @@ jobs:
         # Source helper functions
         . ./.github/scripts/installer-test-helpers.sh
 
+        # Backup contents of existing .bash_profile (if any)
+        [ -e ~/.bash_profile ] && cp ~/.bash_profile ~/.dummy_bash_source
+
+        # Remove existing .bash_profile (file or symlink)
+        rm -f ~/.bash_profile
+
+        # Create symlink pointing to dummy source
+        ln -s ~/.dummy_bash_source ~/.bash_profile
+
```

Comment:
This is preparing the test case; you should not add an empty line between them. Instead, write a comment indicating your intended action and combine these three lines.